portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +27,93 @@ function addFunFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fun_fact;
 }
+
+let count;
+let timer;
+
+/*
+ * Displays the intital image and description once the page is loaded.
+ */
+function initialDisplay() {
+    count = -1;
+    start();
+}
+
+/*
+ * Changes the image and description displayed on the page.
+ */
+function autoSlideshow() {
+    let experiences = document.getElementsByClassName('experience');
+    console.log(experiences);
```

Comment:
Once you are satisfied that the logic works, recommend getting rid of the log statements -- here and elsewhere in Javascript code.

Reply:
Done! (see commit "Address requested changes...")